### PR TITLE
Update and fix payloads

### PR DESCRIPTION
**Pull request by thehappydinoa**
Fixed Style Issues on extensions and payloads.
Added GET TARGET_OS to get.sh
Removed and Fixed all uses ducky_helper.sh (Issue #248)
Removed all mention of DUCKY_LANG (Issue #248)
Renamed Payloads with spaces in name
Added an extension to keep Macs Happy
Added a payload for Mac DNS poisoning
Fixed Issue #271 changed wget to curl -o
Implemented PR #268 
Implemented PR #273 

Special Thanks to @sundhaug92 for checking all my work.

### CODE REVIEW

File: payloads/library/Incident_Response/Hidden_Images/payload.txt [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 ATTACKMODE HID STORAGE
 
-
-DUCKY_LANG gb
-LED B
+LED ATTACK
 RUN WIN powershell -executionpolicy Bypass ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\run.ps1')"
-LED G FAST
-#Green means good to go
+LED FINISH
```
Comment (sebkinne) [inline payloads/library/Incident_Response/Hidden_Images/payload.txt]: This should probably be LED ATTACK or similar.

File: payloads/library/Incident_Response/Link_File_analysis/payload.txt [hunk 2]
```diff
@@ -1,12 +1,7 @@
...
+GET SWITCH_POSITION
 
-
-DUCKY_LANG gb
-LED B
+LED ATTACK
 RUN WIN powershell -executionpolicy Bypass ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\run.ps1')"
-LED G FAST
-#Green means good to go
+LED FINISH
```
Comment (sebkinne) [inline payloads/library/Incident_Response/Link_File_analysis/payload.txt]: This should probably be LED ATTACK or similar.

File: payloads/library/android/fireytv/payload.txt [hunk 3]
```diff
@@ -11,8 +11,13 @@
 # Blue Blinking ...............Running ADB command to push payload.apk
 # Red Blinking.......FireTV failed to get an IP address from the Bash Bunny
 # Green..............Finished
+
+LED SETUP
+GET TARGET_HOSTNAME
```
Comment (sebkinne) [inline payloads/library/android/fireytv/payload.txt]: Should this be GET TARGET_IP?

File: payloads/library/execution/psh_DownloadExec/payload.txt [hunk 4]
```diff
@@ -8,51 +8,49 @@
...
-# Start web server
+# START HTTP SERVER
 iptables -A OUTPUT -p udp --dport 53 -j DROP # disallow outgoing dns requests so server starts immediately
-/usr/bin/gohttp -p 80 -d $PAYLOAD_DIR  > $SERVER_LOG 2>&1 &
+/tools/gohttp/gohttp -p 80 -d /tmp/ > ${SERVER_LOG} 2>&1 &
 
-# Check for psh.txt
-if [ ! -f $PAYLOAD_DIR/psh.txt ]; then
+# CHECK FOR POWERSHELL
+if [ ! -f ${PAYLOAD_DIR}/p.txt ]; then
```
Comment (sebkinne) [inline payloads/library/execution/psh_DownloadExec/payload.txt]: You renamed the file in the code, but the actual psh.txt has not been renamed to p.txt

**Comment by sebkinne** [comment]
I have requested a couple of changes, but this is great, thank you!

File: payloads/library/Incident_Response/Hidden_Images/payload.txt (hunk 1 above)
Comment (thehappydinoa) [reply payloads/library/Incident_Response/Hidden_Images/payload.txt]: @sebkinne thank you for reviewing my work. I will make the changes right now.

**Comment by thehappydinoa** [comment]
@sebkinne I fixed all the issues mentioned in a6ef1bb

**Comment by sebkinne** [comment]
Thanks a lot @thehappydinoa !

**Comment by ibrahimssa** [comment]
Hi, I've tested in on my Mac but the complete top taskbar turned grey and my dock has disappeared since. Any way to fix this.

**Pull request merged**